cypress/e2e/dummy.spec.ts

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+it("Dummy test to make CI pass", () => {});
```

Comment:
I was just going to comment that this was just to make the CI pass for this PR. Without this, the cypress tests would fail because there's no spec files. I'll remove this spec file and other cypress related files in a separate PR.